code/src/reducers/tasks.js

```diff
@@ -0,0 +1,49 @@
+import {createSlice} from '@reduxjs/toolkit'
+
+const taskList = [
+    {id: 1, text: 'My first todo', complete: true, date: new Date().getTime(), due: null},
+    {id: 2, text: 'My second todo', complete: false, date: new Date().getTime(), due: null},
+    {id: 3, text: 'My third todo', complete: true, date: new Date().getTime(), due: null},
+    {id: 4, text: 'My fourth todo', complete: false, date: new Date().getTime(), due: null}
+]
+
+
+
```

Comment:
Some excessive spacing here and there in your whole project - remember to be consistent with spacing as well